src/ui/public/agg_types/buckets/terms.js

```diff
@@ -92,9 +93,14 @@ export default function TermsAggDefinition(Private) {
           $scope.$watch('responseValueAggs', updateOrderAgg);
           $scope.$watch('agg.params.orderBy', updateOrderAgg);
 
+          $scope.rejectAgg = function (agg) {
+            // aggFilter elements all starts with a '!'
+            // so the index of agg.type.name in a filter is 1 if it is included
+            return Boolean(aggFilter.find((filter) => filter.indexOf(agg.type.name) === 1));
+          };
```

Comment:
Ah, I see now. I thought the `aggFilter` variable wasn't getting used at all outside of `rejectAgg`, but now I see it's used when creating a custom order agg as well. This looks fine then.